dialogs/OPTDef.hpp

```diff
@@ -0,0 +1,4 @@
+#define OPT_ColorBack		{0.1,0.1,0.1,1}
```

Comment:
Gilt für alle defines: Der OPT-prefix ist unnötig und wir hatten gesagt, dass alle defines komplett in Großbuchstaben geschriebenw erden solten.

Dh. ich würde mir zB sowas hier vorstellen: `COLOR_BLACK`